### PR TITLE
file_data.py: Add checksums property

### PR DESCRIPTION
**Pull request by PrajwalM2212**
This PR adds checksums property to the FileData class.
It adds the add_checksums method to add new checksum tuples.
The tests have been added for the changes made.

Closes https://github.com/vmware/tern/issues/591

Signed-off-by: PrajwalM2212 <prajwalmmath@gmail.com>

### CODE REVIEW

File: tern/classes/file_data.py [hunk 1]
```diff
@@ -185,6 +196,7 @@ def fill(self, file_dict):
             authors: <authors>
             packages: <packages>
             urls: <urls>
+            checksums: <checksums>
```
Comment (nishakm) [inline tern/classes/file_data.py]: Can you modify the `testFill` test to see if you can fill in a list of checksum tuples?

**Comment by nishakm** [comment]
@PrajwalM2212 will you be updating this PR?

**Comment by PrajwalM2212** [comment]
@nishakm Yes. I have been a little busy lately. But I will push the changes today. Sorry for the delay.

**Comment by PrajwalM2212** [comment]
@nishakm I have pushed the changes.  
GH actions is broken.  CircleCI is working though.

**Comment by nishakm** [comment]
Looks good.
@rnjudge The tests for GHA are failing because of a bad ubuntu mirror
`Could not connect to azure.archive.ubuntu.com:80 (52.177.174.250)`
Merging this.

**Pull request merged**